docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -56,13 +56,124 @@ This makes some tasks more difficult to complete interactively.
 Setting up an Interactive Sim
 -----------------------------
 
-.. todo::
+Setting up an interactive sim for the first time can be confusing. Follow these 
+instructions along with the example notebook included for help. 
+Another useful resource is the `Vivarium InteractiveContext documentation <https://vivarium.readthedocs.io/en/latest/api_reference/interface/interactive.html?highlight=InteractiveContext#vivarium.interface.interactive.InteractiveContext>`_. 
+
+**Example notebook**
+
+This `notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_sim_example_setup.ipynb>`_ includes all of the steps seen below using the CVD project as an example. 
+
+**Step 1: General Prep**
+
+  - Ensure that you have cloned the engineer's GitHub repo and have pulled the latest version of the simulation.  
+  - Check that you have 'vivarium' installed in your environment and can import 'InteractiveContext' in a Jupyter notebook. 
+  - If you need more general help with Git or environments for this part, see these :ref:`computing resources <computing>`.
+
+Example code: 
+
+:: 
+
+  from vivarium import InteractiveContext 
+
+
+**Step 2: Creating a Simulation** 
+
+Now that you're prepped and able to run a simulation, we can actually load the sim! To 
+do this, we will use the 'InteractiveContext' function in Vivarium. 
+
+First, you'll need to locate the model specs file. It will be in the engineering repo usually 
+in a file path similar to: :code:`<project_repo_name>/src/<project_repo_name>/model_specifications/`. 
+It will be a .yaml file. 
+
+If you can't locate the proper model specs file or are unsure - ask the engineers for help! 
+
+Once you have that, you can follow this code to create a simulation 
+
+::
+
+  path = Path('<file_path_for_model_spec_file>')
+  
+  sim = InteractiveContext(Path(path), setup=False)
```

Comment:
```suggestion
  sim = InteractiveContext(path, setup=False)
```
I don't think you need to make it into a path here, it is already a path from the previous line. (But please confirm that this works!)